portal/src/store/modules/materials.js

```diff
@@ -180,6 +181,20 @@ export default {
       } else {
         $log.error('Validate error: ', external_id)
       }
+    },
+    async getSimilarMaterials(context, { external_id, language }) {
+      if (validateIDString(external_id)) {
+        return await axios
```

Comment:
Als we toch async/await gebruiken dan vind ik het persoonlijk vaak leesbaarder om alleen dat te gebruiken, dus:

```
const res = await axios.get('suggestions/similarity/', {
            params: {
              external_id: external_id,
              language: language
            }
          })

return res.data
```